lib/web/app/redirect.go

```diff
@@ -60,3 +62,59 @@ func MetaRedirect(w http.ResponseWriter, redirectURL string) error {
 	SetRedirectPageHeaders(w.Header(), "")
 	return trace.Wrap(metaRedirectTemplate.Execute(w, redirectURL))
 }
+
+const appRedirectHTML = `
```

Comment:
Consider something similar to the line above

https://github.com/gravitational/teleport/blob/de6bf60dce11e0051161d5044797a2d4354840cc/lib/web/app/redirect.go#L43

such that this template doesn't need to be re-parsed each time the respective endpoint is hit.